sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 12.0.0-beta.2 (Unreleased)
+## 12.0.0-beta.3 (Unreleased):
+
+## 12.0.0-beta.2 (2020-10-06):
 
 ### Added
```

Comment:
Also, for the changelog sections, we need to make sure to follow this https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change